src/main/resources/templates/start.html

```diff
@@ -26,35 +26,7 @@
     <title>Home</title>
 </head>
 <body onload="bindClickHandlers()">
-    <nav class="navbar navbar-expand-lg navbar-light bg-light">
```

Comment:
I removed this duplicate code and used a thymeleaf tag to import the navbar from /fragments/navbar.html to adhere to clean code standards and best practices that code should never repeat itself. Read more about best practices here: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3